Code/Engine/RendererCore/RenderContext/Implementation/RenderContext.cpp

```
@@ -853,6 +855,56 @@ ezGALSamplerStateHandle ezRenderContext::GetDefaultSamplerState(ezBitflags<ezDef
 // private functions
 //////////////////////////////////////////////////////////////////////////
 
+// static
+void ezRenderContext::LoadBuiltinShader(ezShaderUtils::ezBuiltinShaderType type, ezShaderUtils::ezBuiltinShader& out_shader)
+{
+  ezShaderResourceHandle hActiveShader;
+  switch (type)
+  {
+    case ezShaderUtils::ezBuiltinShaderType::CopyImage:
+      hActiveShader = ezResourceManager::LoadResource<ezShaderResource>("Shaders/Pipeline/Copy.ezShader");
+      break;
+    case ezShaderUtils::ezBuiltinShaderType::DownscaleImage:
+      hActiveShader = ezResourceManager::LoadResource<ezShaderResource>("Shaders/Pipeline/Downscale.ezShader");
+      break;
+  }
+
+  EZ_ASSERT_DEV(hActiveShader.IsValid(), "Could not load builtin shader!");
+
+  if (!hActiveShader.IsValid())
+    return;
```

Output an error instead of silently returning?